File: JAVA/baidumap-webapi/src/test/java/wang/seamas/baidumap/CoordTransformTest.java
package wang.seamas.baidumap;

import org.junit.Assert;
import org.junit.Test;
import wang.seamas.baidumap.request.CoorTransformRequest;
import wang.seamas.baidumap.request.model.CoordTransfModel;
import wang.seamas.baidumap.response.CoordTransformResponse;

/**
 * @author: Seamas Wang
 * @description:
 * @date: Created in 12:34 2018/8/6
 */
public class CoordTransformTest {

    @Test
    public void test() {
        BaiduApiClient client = ClientCreator.Create();

        CoordTransfModel model = new CoordTransfModel();
        model.setCoords("114.21892734521,29.575429778924");

        CoorTransformRequest request = new CoorTransformRequest(model);
        CoordTransformResponse response = client.execute(request);
        Assert.assertEquals(0, response.getStatus());
    }
}

File: DOTNET/BaiduMapTest/CoordTransformTest.cs
using System;
using BaiduMap.Request;
using BaiduMap.Request.Models;
using Xunit;
using Shouldly;

namespace BaiduMapTest
{
    public class CoordTransformTest
    {

        [Fact]
        public void Test() 
        {
            var client = ClientCreator.Create();

            var model = new CoordTransfModel
            {
                Coords = "114.21892734521,29.575429778924"
            };

            var req = new CoordTransformRequest(model);
            var resp = client.Execute(req);

            resp.Status.ShouldBe(0);
        }

    }
}
